common/mail.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable max-classes-per-file */
+class Mail {
+  /**
+   *
+   * @param {Object}
+   *
+   */
+  constructor({
+    subject, to, html, attachment,
+  }) {
+    this.from = 'Easy Event <admin.easy-event@mail.namdaoduy.dev>';
```

Comment:
Move to constant

Reply:
`Email = {
name:
address:
}`

Reply:
Solved: https://github.com/EasyEventV2/ee2_api/pull/9/commits/409cebfa64b084cb7802c255ca941b8fb38eb2ed